lib/Backend/BackwardPass.cpp

```diff
@@ -2965,6 +2974,43 @@ BackwardPass::DeadStoreOrChangeInstrForScopeObjRemoval()
     return false;
 }
 
+IR::Instr *
+BackwardPass::TryChangeInstrForStackArgOpt()
+{
+    IR::Instr * instr = this->currentInstr;
+    if (instr->DoStackArgsOpt(this->func))
+    {
+        switch (instr->m_opcode)
+        {
+            case Js::OpCode::TypeofElem:
+            {
+                /*
+                    Before:
+                        dst = TypeOfElem arguments[i] <(BailOnStackArgsOutOfActualsRange)>
```

Comment:
Yes. You are right. But "before" here doesn't indicate today's code, but it indicates the current state of the code and how it gets transformed.

---
In reply to: [70345281](https://github.com/Microsoft/ChakraCore/pull/1261#discussion_r70345281) [](ancestors = 70345281)